docs/pages/versions/unversioned/sdk/linear-gradient.mdx

```diff
@@ -79,4 +79,10 @@ const styles = StyleSheet.create({
 import { LinearGradient } from 'expo-linear-gradient';
 ```
 
+For TS to know the provided array has 2 or more values, it needs to be provided "inline" or typed `as const`. For example:
```

Comment:
can you move this directly into the docs for the `color` prop? It'll be better positioned there I think. They are generated from packages/expo-linear-gradient/src/LinearGradient.tsx